zerver/fixtures/bugdown-data.json

```diff
@@ -421,6 +421,24 @@
       "bugdown_matches_marked": true
     },
     {
+      "name": "timestamp",
+      "input": "!time(Jun 5th 2017, 10:30PM)",
+      "expected_output": "<p><span class=\"timestamp\" value=\"1496701800\"><img alt=\":clock4:\" class=\"emoji\" src=\"/static/generated/emoji/images/emoji/unicode/1f553.png\" title=\":clock4:\"><span value=\"1496701800\">Jun 5th 2017, 10:30PM</span></span></p>",
+      "bugdown_matches_marked": false
+    },
```

Comment:
Because this particular date is not in a format that moment supports, whereas the dateutils module supports this. Thus, we have a mismatch. I think I should add a simple test case in a format that's valid on both the systems, as well as keep this one around.